CK3ToEU4Tests/MapperTests/StartDateMapper/StartDateMapperTests.cpp

```diff
@@ -0,0 +1,105 @@
+#include "../../CK3ToEU4/Source/Mappers/StartDateMapper/StartDateMapper.h"
+#include "gtest/gtest.h"
+#include <sstream>
+
+TEST(Mappers_StartDateMapperTests, startDateDefaultsToDefault)
+{
+	std::stringstream input;
+
+	const mappers::StartDateMapper theMapper(input);
+
+	ASSERT_TRUE(theMapper.getStartDate().isSet());
+	EXPECT_EQ(theMapper.getStartDate().getYear(), 1444);
+	EXPECT_EQ(theMapper.getStartDate().getMonth(), 11);
+	EXPECT_EQ(theMapper.getStartDate().getDay(), 11);
+}
+
+TEST(Mappers_StartDateMapperTests, startDateCanBeSet)
+{
+	std::stringstream input;
+	input << "year = 1425\n";
+	input << "month = 7\n";
+	input << "day = 19\n";
+
+	const mappers::StartDateMapper theMapper(input);
+
+	ASSERT_TRUE(theMapper.getStartDate().isSet());
+	EXPECT_EQ(theMapper.getStartDate().getYear(), 1425);
+	EXPECT_EQ(theMapper.getStartDate().getMonth(), 7);
+	EXPECT_EQ(theMapper.getStartDate().getDay(), 19);
+}
+
+TEST(Mappers_StartDateMapperTests, invalidYearBailsToDefault)
```

Comment:
If you wanted to be really thorough, you could test too big and too small for each of these.